alertwildfire_live_recording.py

```diff
@@ -146,28 +130,39 @@ def fetchAllCameras(googleServices, camera_names_to_watch):
 def main():
     """directs the funtionality of the process ie start a cleanup, record all cameras on 2min refresh, record a subset of cameras, manage multiprocessed recording of cameras
     Args:
-        -c  cleaning_threshold" (flt): time in hours to store data
-        -o  cameras_overide"    (str): list of specific cameras to watch
+        -o  cameras_overide    (str): list of specific cameras to watch
+        -p  parallelize       (bool): toggle to parallelize
     Returns:
         None
     """
     reqArgs = []
     optArgs = [
-        ["c", "cleaning_threshold", "time in hours to store data"],
-        ["o", "cameras_overide", "specific cameras to watch"]
+        ["o", "cameras_overide", "specific cameras to watch"],
+        ["p", "parallelize", "toggle parallelisation"]
     ]
     args = collect_args.collectArgs(reqArgs,  optionalArgs=optArgs, parentParsers=[goog_helper.getParentParser()])
-    googleServices = goog_helper.getGoogleServices(settings, args)
-    if args.cleaning_threshold:
-        cleaning_threshold = float(args.cleaning_threshold)
-        cleanup_archive(googleServices, cleaning_threshold)
+    
     if args.cameras_overide:
         listofRotatingCameras = list(args.cameras_overide.replace(" ", "").strip('[]').split(','))
     else:
         listofCameras = alertwildfire_API.get_all_camera_info()
         listofRotatingCameras = [camera["name"] for camera in listofCameras if (camera["name"][-1]=='2') ]
+    if args.parallelize:
+        parallel = args.parallelize
+    else:
+        parallel = False
 
-    fetchAllCameras(googleServices, listofRotatingCameras)
+    if parallel:#having issues
+        num_cameras_per_process = 5
```

Comment:
Undocumented fixed magic numbers are usually not great.  Either move these as command line arguments or document how you arrived at these values.